eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/OfflineStateSpec.scala

```diff
@@ -508,9 +504,23 @@ class OfflineStateSpec extends TestkitBaseClass with StateTestsHelperMethods {
     val highFeerate = FeeratesPerKw.single(tooHighFeeratePerKw)
 
     // this time Alice will ignore feerate changes for the offline channel
```

Comment:
nit: both the name of the test and this comment are outdated now, i think either we split the test or update the name to something more descriptive.

Reply:
Right, the comment should be updated to say that we only handle feerate change when reconnecting, thanks